packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -31,12 +32,14 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
   private _stickyContentTop = createRef<HTMLDivElement>();
   private _stickyContentBottom = createRef<HTMLDivElement>();
   private _nonStickyContent = createRef<HTMLDivElement>();
+  private _placeHolder = createRef<HTMLDivElement>();
 
   constructor(props: IStickyProps) {
     super(props);
     this.state = {
       isStickyTop: false,
-      isStickyBottom: false
+      isStickyBottom: false,
+      distanceFromTop: 0
```

Comment:
Sticky also has a public member variable distanceFromTop. This is bound to create confusion. Please remove that one.

Since it is used outside, we may have to expose a public getter for this.

Reply:
ScrollablePane is using distanceFromTop to sort Stickies. This would not be possible if its read from the Sticky's state.